src/styles/typography/index.md.njk

```diff
@@ -0,0 +1,120 @@
+
+You can also add classes to create lead paragraphs and smaller body copy to convey hierarchy in your page.
+
+### Lead paragraph
+
+A lead paragraph is an introductory paragraph that you can use to summarise the content of the page below. Lead paragraphs use 24px type on desktop and should only be used once per page if needed.
+
+{{ example({group: 'styles', item: 'typography', example: 'lead', html: true, nunjucks: true, open: false}) }}
+
+### Body small
+
+You can use the body small class to make your paragraph font size smaller: 16px on larger screens and 14px on smaller screens.
+
+Generally, you should only use body small for supporting content like the ‘Related items’ navigation on GOV.UK. Use body small sparingly.
+
+The majority of your body copy should use the standard 19px paragraph size.
+
+{{ example({group: 'styles', item: 'typography', example: 'small', html: true, nunjucks: true, open: false}) }}
+
+## Font override classes and typography mixins
+
+You might need to set the font size or font weight of an element outside of the predefined heading and paragraph classes. For this you can use the font utility classes in your HTML or reference the typography mixins in your own components.
+
+### Font size
+
+The full GOV.UK typography scale goes from 14px up to 80px on large screens. You can add these font-size override classes to any other typographic class or element and they will change the font size.
+
+{{ example({group: 'styles', item: 'typography', example: 'font-size', html: true, nunjucks: true, open: false}) }}
+
+### Font weight
+
+As with the font size, you can add a font-weight override class to any other typographic class or element to change the font weight to regular or bold weight.
+
+{{ example({group: 'styles', item: 'typography', example: 'font-weight', html: true, nunjucks: true, open: false}) }}
+
+## Links
+
+Links are blue and underlined by default. If your link is at the end of a sentence or paragraph, make sure that the linked text doesn’t include the full stop.
+
+{{ example({group: 'styles', item: 'typography', example: 'link', html: true, nunjucks: true, open: false}) }}
+
+## Lists
+
+Use lists to make blocks of text easier to read, and to break information into smaller, manageable chunks.
+
+{{ example({group: 'styles', item: 'typography', example: 'list', html: true, nunjucks: true, open: false}) }}
+
+### Bulleted lists
+
+Introduce bulleted lists with a lead in line ending in a colon. Start each item with a lowercase letter, and don’t use a full stop at the end.
+
+{{ example({group: 'styles', item: 'typography', example: 'list-bullet', html: true, nunjucks: true, open: false}) }}
+
+### Numbered lists
+
+Use numbered lists instead of bulleted lists when the order of the items is relevant.
+
+You don’t need to use a lead-in line for numbered lists. Items in a numbered list should end in a full stop because each should be a complete sentence.
+
```

Comment:
The examples for this do not end in a full stop or aren't a complete sentence.

Reply:
Great spot, thanks!